peps/pep-0760.rst

```diff
@@ -0,0 +1,162 @@
+PEP: 760
+Title: No More Bare Excepts
+Author: Pablo Galindo <pablogsal@python.org>, Brett Cannon <brett@python.org>
+PEP-Delegate: TBD
```

Comment:
We can delete this, most (non-packaging/typing) things go direct to SC. Unless you're planning for a delegate? 
```suggestion
```